src/V3Global.h

```diff
@@ -47,6 +47,7 @@ class V3HierBlockPlan;
 // Constructor needs () or GCC 4.8 false warning.
 #define VL_RESTORER(var) \
     const VRestorer<typename std::decay<decltype(var)>::type> restorer_##var(var);
+#define VL_GET_SAVED(var) restorer_##var.saved()
```

Comment:
```suggestion
#define VL_RESTORER_GET(var) restorer_##var.saved()
```

Or something so named similar to VL_RESTORE...

Reply:
How about `VL_RESTORER_PEEK`

Reply:
VL_RESTORER_PAST?  or _PREV?

Reply:
VL_RESTORER_PREV seems better